daos/stored_recipes/test/stored_recipes_logic_test.py

```diff
@@ -0,0 +1,152 @@
+import unittest
+
+from flask import Flask, jsonify
+
+from daos.database import DataBase
+from daos.stored_recipes import stored_recipes_manager
+
+app = Flask(__name__)
+app.config['URL_TIMEOUT'] = 10
+app.app_context().push()
+with app.app_context():
+    db = DataBase(app)
+    db_instance = db.get_db_instance()
+    stored_recipe_dao = stored_recipes_manager.RecipeService(db_instance)
+    db_instance.drop_all()
+    db_instance.create_all()
+
+recipes_mocks = [
+    {
+        "ingredients": [],
+        "instructions": [],
+        "url": "www.mockito.com",
+        "title": "mockimo"
+    },
+    {
+        "ingredients": [
+            "   לסיר מרק קוסקוס בסגנון מרוקאי",
+            "   10 שוקיים של עוף אני קוטמת להם את הקצה",
+            "   1 קג בשר לבישול ארו  1/4 כוס שמן זית לבישול הבשר 56102 אוסו בוקו",
+            "   2 כוסות גרגירי חומוס גדולים שהושרו לילה במים רותחים  אפשר חומוס מוכן מקופסת שימורים רק לשטוף אותם היטב לפני שמשתמשים",
+            "   5 גזרים ",
+            "   3 תפוחי אדמה",
+            "   56 ענפי סלרי כולל העלים  ",
+            "   1 ראש כרוב חתו ל  6",
+            "   3 בצלים",
+            "   1 פלפל ירוק חריף לא חובה אם לא רוצים חריפות עדינה",
+            "   5 קישואים",
+            "   1 קג דלעת/דלורית",
+            "   2 עגבניות",
+            "   חצי חבילה כוסברה",
+            "   56  עלי דפנה",
+            "   56 גרגרי פלפל אנגלי שלם",
+            "   מלח",
+            "   1 כפית פלפל שחור ",
```

Comment:
nice idea, two things:
under the test module, add this mock recipe in a separate files-class, it's spamming the file.
the tested logic here is storing you don't need it to be so long (be cheap!)

Reply:
Fixed!